opportunities_test_element.rb

```diff
@@ -0,0 +1,32 @@
+class Opportunities_testPage
```

Comment:
Move all these elements and use the existing elements from OpportunitiesPage POM. Delete this file after making the changes in OpportunitePage.